content/ai/concepts/neural-networks/terms/artificial-neural-networks/artificial-neural-networks.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Artificial Neural Networks'
+Description: 'A neural network is a method in artificial intelligence that teaches computers to process data in a way that is inspired by the human brain.'
+Subjects:
+  - 'Machine Learning'
+  - 'Brain Activity'
+  - 'Machine Learning'
+  - 'Deep Learning'
+  - 'Perceptrons'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'ANN'
+  - 'Neural Networks'
+  - 'Weight & Bias'
+  - 'Neurons'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+**Artificial Neural Networks (ANN)** A neural network is a method in artificial intelligence that teaches computers to process data in a way that is inspired by the human brain. 
+## What Is An Artificial Neural Network (ANN)
+
+- It is a type of machine learning process, called deep learning, that uses interconnected nodes or neurons in a layered structure that resembles the human brain.
+- An ANN is one of the main tools used for machine learning, they are brain-inspired systems used to replicate the way that we humans learn.
+
+### The Process
+
+- Neural Networks are trained on specific data which then get broken down into the input layer ( where information is fed into) then to the hidden layer (where weight & bias are calculated from the input layer) & lastly the output layer ( where information is output after algorithms are processed from the hidden & input layers) via perceptrons (similar to human neurons but for machines to process, store, calculate & move data forward)
```

Comment:
```suggestion
- Neural networks are trained on specific data which then get broken down into the input layer (where information is fed into) then to the hidden layer (where weight & bias are calculated from the input layer) and lastly the output layer (where information is output after algorithms are processed from the hidden and input layers) via perceptrons (similar to human neurons but for machines to process, store, calculate & move data forward)
```

Reply:
done but what about the & between (weight & bias)?

Reply:
also (calculate & move data)?

Reply:
@DevGat718 Yes, I missed those.

Reply:
copy im on it. @SSwiniarski